src/pages/AllPrograms.jsx

```diff
@@ -4,37 +4,37 @@ import { gql, useQuery } from "@apollo/client";
 
 // TODO Es werden nicht alle Programme direkt geladen. Entweder baut ihr ein endless scrolling wo ab einer bestimmten Scrollposition neue Programme geladen werden oder einen "weitere laden" Button unter den jeweils schon angezeigten Programmen
 
-// TODO get this working
 const GET_PROGRAMS = gql`
   query GetPrograms {
     programs {
       name
     }
   }`;
 
-  function Programs({ GetPrograms }) {
-    const { loading, error, data } = useQuery(query);
+
+export default function AllPrograms() {
+  const { loading, error, data } = useQuery(GET_PROGRAMS);
   
-    if (loading) return null;
-    if (error) return `Error! ${error}`;
+  if (loading) return null;
+  if (error) return `Error! ${error}`;
   
-    return <pre>{JSON.stringify(data, null, 2)}</pre>;
-  }
-
 
-export default function AllPrograms() {
   return (
     <DefaultLayout>
       <h2 className="py-14 pl-4 text-left text-3xl font-bold text-light">
         Browse
       </h2>
-      {/* TODO make gradients alternate */}
-      <ProgramMedium>
-        {/* TODO add data from GraphQL */}
-        <p>Test</p>
+      {/* TODO make gradients alternate and get all the programs without hardcoding it here ... oder ist das zu kompliziert gedacht?*/}
```

Comment:
Instead connecting the programs of hygraph with gradients (or the other way around) you could keep the gradient order (Orange, Green, Blue, Orange... from up to down) and insert the data of programs despite of the color.